ui/json_ui.go

```diff
@@ -134,18 +134,25 @@ func (ui *jsonUI) Flush() {
 	}
 }
 
-func (ui *jsonUI) stringRows(header []Header, rows [][]Value) []map[string]string {
-	result := []map[string]string{}
+func (ui *jsonUI) stringRows(header []Header, rows [][]Value) []map[string]interface{} {
+	result := []map[string]interface{}{}
 
 	for _, row := range rows {
-		data := map[string]string{}
+		data := map[string]interface{}{}
 
 		for i, col := range row {
-			if header[i].Hidden {
+			if header[i].Hidden || col.String() == "" {
 				continue
 			}
 
-			data[header[i].Key] = col.String()
+			switch v := col.(type) {
```

Comment:
instead of doing it based on concrete type, let's add `Raw() interface{}` to each value type, then here we can just cast to this interface. if not castable to interface, fallback to String().